lib/ci-helper.ts

```diff
@@ -257,24 +260,15 @@ export class CIHelper {
             return false;
         }
         let result = false;
-        let optionsUpdated = false;
         for (const pullRequestURL in options.openPRs) {
             if (!Object.prototype.hasOwnProperty.call(options.openPRs, pullRequestURL)) {
                 continue;
             }
             console.log(`Handling ${pullRequestURL}`);
-            const [notesUpdated, optionsUpdated2] = await this.handlePR(pullRequestURL, options);
-            if (notesUpdated) {
+            const [notesUpdated, optionsUpdated] = await this.handlePR(pullRequestURL, options);
+            if (notesUpdated || optionsUpdated) {
```

Comment:
The jsdoc return description no longer needs to specify `(and needs to be pushed)`.

Reply:
Fixed and pushed.